### DISCUSSION TITLE
Database schema validation failed

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#1990] [Q&A]
I am getting this error.

[error](https://github.com/quartznet/quartznet/blob/main/src/Quartz/Impl/AdoJobStore/JobStoreSupport.cs#L550)

Version: 3.5

I have these tables created in my sql server database.

![image](https://user-images.githubusercontent.com/3524495/228940510-e5caf77f-1334-4b3e-8f7e-810d20e84fa6.png)

What could be the issue?

FYI, I have different databases for each tenant. It's working fine for other tenant.

**Comment by lahma** [comment]
https://stackoverflow.com/questions/75892425/quartz-database-schema-validation-failed

**Reply by viveknuna** [reply]
@lahma that is my question only, I didn't get any help there.

**Reply by lahma** [reply]
You do realize that cross-posting questions between services without notifying and possibly removing duplicate work is like the most frowned upon thing in OSS?

**Comment by kevingy** [comment]
I'm getting the same error message in 3.8. The inner exception is `Object cannot be cast from DBNull to other types.`, but there is no indication of what object is null or being cast.
